executor/containerdexecutor/executor.go

```diff
@@ -79,27 +79,23 @@ func (w containerdExecutor) Exec(ctx context.Context, meta executor.Meta, root c
 		lm.Unmount()
 	}
 
-	var iface network.Interface
 	// FIXME: still uses host if no provider configured
 	if meta.NetMode == pb.NetMode_UNSET {
-		if w.networkProvider != nil {
-			var err error
-			iface, err = w.networkProvider.NewInterface()
-			if err != nil || iface == nil {
-				meta.NetMode = pb.NetMode_HOST
-			}
-		} else {
-			meta.NetMode = pb.NetMode_HOST
-		}
+		meta.NetMode = pb.NetMode_HOST
```

Comment:
@crosbymichael you need to remove this if here, as callers of buildkit may want to set other things for UNSET. I would do the override somewhere else that's specific to `buildkitd`